src/main/java/com/BrewMES/demo/model/BrewMES.java

```diff
@@ -37,7 +38,7 @@ public void getReport(Batch batch) {
 		throw new UnsupportedOperationException();
 	}
 
-	public void connectMachine(String ipAddress) {
+	public UUID connectMachine(String ipAddress) {
```

Comment:
Undocumented change. Why are we now returning `UUID` instead of `void`?

Reply:
Brainfart. I removed it -> https://github.com/Semester3-03/Java-application/pull/72/commits/abd524e8cb3c24bec21c4030ef7d3540e60929ea